src/js/popup.js

```diff
@@ -348,6 +360,48 @@ function deactivateOnSite() {
   });
 }
 
+/**
+ * Open the share overlay (TODO currently an alert)
+ */
+function share() {
+  $("#share_overlay").toggleClass('active');
+  var shareMessage = chrome.i18n.getMessage("share_base_message");
+
+  //Only add language about found trackers if we actually found trackers (but regardless of whether we are actually blocking them).
+  if (!POPUP_DATA.noTabData) {
```

Comment:
We should refactor for readability to short circuit (end function early) instead of continually nesting the code.